docs/PR_GUIDE.md

```diff
@@ -0,0 +1,92 @@
+- All commits and PR captions should be written in English. PR description and PR comments may be written in other
+  languages
+- All PRs should be approved by at least two reviewers. There are some exceptions: (1) simple PRs to some tools or tests;
+  (2) situation when it’s impossible to find another competent reviewer. In any case a PR should be reviewed by
+  at least one reviewer
+- An author should not merge their PR. One of the reviewers should merge it
+- If somebody left a comment in PR the author should wait for their approval
+
+
+## Doing good code review:
+
+- All comments in PR should be polite and concern the PR
+- If a reviewer criticizes a PR they should suggest a better solution
+- Reviewer's solution may be rejected by the author if the review did not give arguments why it should be done 
+- Comments in PR should be not only about things which are worth redeveloping but about good designs as well
+- It’s better to ask developer to make code simpler or add comments to code base than to understand the code through
+  developer's explanation
+- It’s worth writing comments in PR which help a PR author to study something new
+- All code base should conform ./omim/docs/CPP_STYLE.md, ./omim/docs/OBJC_STYLE.md or other style in ./omim/docs/
+  depending on the language
+- Code review should be started as quick as possible after a PR is published. A reviewer should answer developer
+  comments as quick as possible
+- If a PR looks good to a reviewer they should approve this PR. It means the review is finished
+- If a developer changes the PR significantly they should inform reviewers, who approved the PR about these changes
+- A reviewer should note not only to code base but also to the description of the PR and commits
+- A PR may be merged by a reviewer if all the following items are fulfilled: (1) the PR isn't marked as a draft or [DNM];
+  (2) all reviewers which have started to review the PR, approved it; (3) all reviewers which are added as reviewers
+  of the PR, have approved it
+- If a reviewer doesn’t have time to review all the PR they should write about it explicitly. For example, LGTM for
+  android part
+- If a reviewer and a developer cannot find a compromise, the third opinion about it should be asked
+- If a reviewer is off but a PR should be merged urgently, the following steps should be done by the author: (1) try to find
+  the reviewer and ask to finish the review; (2) ask CIO for a «gold» LGTM which let merge the PR without waiting for
+  reviewers
+- All comments about blank lines should be considered as optional
+
+
+## Recommendations:
+
+- Functions and methods should not be long. In most cases it’s good if the whole body of a function or method fits on
+  the monitor. It’s good to write function or method shorter than 60 lines
+- Source files should not be very long
+- If you solve a big task it’s worth splitting it into subtasks, then making in jira an epic for the big task and
```

Comment:
s/Then developing/Then develop/

Reply:
`- If you are solving a big task it’s worth splitting it into subtasks, then making in jira an epic for the big task and  tickets in the epic for subtasks. Then develop one or several PRs for every task`